test/jasmine/tests/gl2d_plot_interact_test.js

```diff
@@ -789,4 +789,47 @@ describe('@gl Test gl2d plots', function() {
         .catch(fail)
         .then(done);
     });
+
+    it('should be able to draw more than 4096 colors', function(done) {
+        var gd = createGraphDiv();
+        var x = [];
+        var color = [];
+        var N = 1e5;
+        var w = 500;
+        var h = 500;
+
+        for(var i = 0; i < N; i++) {
+            x.push(i);
+            color.push(Math.random());
```

Comment:
maybe I'm being paranoid, but I still don't like `random` in tests. Can we do something repeatable like https://github.com/plotly/plotly.js/blob/ce5bc971770daaf3ed60d374f23a8af1bb61fe0b/src/traces/box/plot.js#L17-L30?

Reply:
Ha right. Good call!

Reply:
done via 
20c3a00 and eae6da2